Announce membership programme (#1875)
matrix-org/matrix.org: content/blog/2023/06/2023-06-20-membership-program.md

communications, Matrix brings the sovereignty, security and interoperability,
which are key to having full control over one’s own communication. By
financially supporting the project, people who are aligned with our beliefs
allow us to keep fighting for our collective rights, be it by providing secure
software or persuading regulators to protect their citizens (such as our
[DMA promotion work](https://matrix.org/blog/2023/03/15/the-dma-stakeholder-workshop-interoperability-between-messaging-services/)
or our [fight against the Online Safety Bill](https://matrix.org/blog/2021/05/19/how-the-u-ks-online-safety-bill-threatens-matrix)).

**A curated protocol benefits profit-making companies building communication
products and services.** Decentralisation is a hard problem. End-to-end
encryption is a hard problem. Decentralised end-to-end encrypted communications
are a very hard problem. Matrix is doing the heavy lifting in these areas so
companies can focus on building what they do best: creating great user
experiences on top of it. By financially supporting the project, organisations
building on Matrix rest assured it keeps evolving, being patched and staying

> secure. In short it ensures Matrix remains a competitive advantage over other

Reviewer: "remainds advantaged" ("competitively"?) or, easier:

```suggestion
secure. In short it ensures Matrix maintains a competitive advantage over other
```

another alternative:

```suggestion
secure. In short it ensures Matrix keeps a competitive advantage over other
```

Author: I mean to say using Matrix is a competitive advantage, I'm not sure your suggestions keep that meaning

Reviewer: No it doesn't, but it was not clear to me at all that that is what this was trying to say.

products, at a fraction of what it would cost to maintain an in-house solution.

**A curated protocol benefits entire governments, and large parts of the public
sector**, who have both adopted it widely. Public organisations can not only
benefit from a sovereign, secure and interoperable solution - they can also
ensure public money is spent for public good, not shareholder wealth. By joining
the Matrix Foundation, public sector entities contribute to the stability,
security and performance of Matrix. Investing into the project ensures that the
open standard that is supporting their healthcare, defence and public
administration continues to benefit from innovative open source software
development.

## Stay unbiased & vendor neutral

> The Matrix manifesto hasn’t changed. We still believe people should have full

Reviewer: Should we use regular ASCII apostrophes, or are the ones you used somehow preferable?

control over their own communication, not be locked in silos, be able to
converse securely and privately, and that communication should be available to
everyone as a free and open, unencumbered, standard and global network.

> The Foundation maintains the Spec, which formalises the behaviour expected from

Reviewer: ```suggestion
The Foundation maintains the Matrix Specification, which formalises the behaviour expected from
```

the various software components of the Matrix ecosystem. We believe it’s
important for both individuals using the protocol - and organisations building

> products on top of it - to be part of the decision process when it comes to

Reviewer: ```suggestion
products on top of it - to be part of the decision-making process when it comes to
```

Small style nit, "decision-making process" seems like a very common term while I don't think I've seen "decision process" before.

> shaping the evolution and functionalities of Matrix. So we have designed

Reviewer: "functionalities" also seems like a very uncommon word, would "features" match the intended meaning?

recommendations. We are [hiring a Managing Director](https://apply.workable.com/elementio/j/0D7080B8A1/)
who will be in charge of identifying and building programmes the Foundation can
deliver, obtaining funding for it, building a team and overseeing the delivery
with the approval and support of the Governing Board.

Such programmes could include an accreditation process, to give more visibility
and credibility to the organisations adopting Matrix seriously, organising more
Matrix events, to promote the standard and share experiences amongst the
community etc. Their mission will overall be to make the ecosystem thrive and
grow.

A Managing Director working full time will also allow us to increase the
Foundation’s transparency by allowing it to report more often on such
programmes, independently from any vendor.

## Going where we’re expected to

The Governing Board is the new compass of the Foundation: it refines the vision
of the Foundation and steers the Managing Director and Spec Core Team in the
right direction.

This gives the Governing Board a lot of power over the Foundation and Matrix,
so we’ve put in place several safeguards to ensure the Foundation remains

> healthy. 

Reviewer: Extra trailing space.
